index.html

```diff
@@ -0,0 +1,200 @@
+                    <h2>Marketers</h2>
+                    <p>Few would argue that, despite the advancements of feminism over the past three decades, women still face a double standard when it comes to their behavior. </p>
+                    <h3>Accessory makers</h3>
+                    <p>While most people enjoy casino gambling, sports betting, lottery and bingo playing for the fun</p>
+                    <h3>Alterationists</h3>
+                    <p>If you are looking for a new way to promote your business that won’t cost you more money, </p>
+                    <h3>Custom Design designers</h3>
+                    <p>If you are looking for a new way to promote your business that won’t cost you more money, </p>
+                </div>
+                <div>
+                    <img src="img/marketers.jpg" alt="Section picture">
+                </div>
+            </section>
+            <section class="section-hero">
+                <div class="container">
+                    <h2 class="center-style">Need a super hero?</h2>
+                    <p class="center-style">Do you require some help for your project: Conception workshop, prototyping, marketing strategy, landing page, Ux/UI?</p>
+                    <button class="yellow-button center-style">Contact our expert</button>
+                    <img src="img/avatar_1.png" alt="Avatar picture">
+                    <img src="img/avatar_2.png" alt="Avatar picture">
+                    <img src="img/avatar_3.png" alt="Avatar picture">
+                    <img src="img/avatar_4.png" alt="Avatar picture">
+                    <img src="img/avatar_5.png" alt="Avatar picture">
+                </div>
+            </section>
+            <section class="section-news container">
+                <h2>Marketing Strategies</h2>
+                <p>Join 40,000+ other marketers and get proven strategies on email marketing</p>
+                <div class="news-card">
+                    <div>
+                        <img src="img/card_image.jpg" alt="Card picture">
+                        <span>By <span class="author-name">Abdullah</span> |  03 March 2019</span>
+                        <h3>Increasing Prosperity With Positive Thinking</h3>
+                    </div>
+                    <div>
+                        <img src="img/card_image_2.jpg" alt="Card picture">
+                        <span>By <span class="author-name">Abdullah</span> |  03 March 2019</span>
+                        <h3 class="active-title">Motivation Is The First Step To Success</h3>
+                    </div>
+                    <div>
+                        <img src="img/card_image_3.jpg" alt="Card picture">
+                        <span>By <span class="author-name">Abdullah</span> |  03 March 2019</span>
+                        <h3>Success Steps For Your Personal Or Business Life</h3>
+                    </div>
+                </div>
+                <div class="carousel-buttons">
```

Comment:
You can use <ul> for this list of dots

Reply:
and you can't put block element inside inline) <a><div></div></a>